src/main/scala/uk/gov/hmrc/nunjucks/NunjucksHelperModule.scala

```diff
@@ -107,13 +136,26 @@ object NunjucksHelper {
       .getThreadLocal("request")
       .asInstanceOf[RequestHeader]
 
-    val reverseRoutes: NunjucksRoutesHelper = cx
+    val routesHelper: NunjucksRoutesHelper = cx
       .getThreadLocal("reverseRoutes")
       .asInstanceOf[NunjucksRoutesHelper]
 
-    val script = JavaScriptReverseRouter("routes")(reverseRoutes.routes: _*)(request).toString
+    val scope = {
+      val context = Context.enter()
+      val scope   = context.initSafeStandardObjects(null, true)
+      Context.exit()
+      scope
+    }
+
+    cx.evaluateString(scope, routesInitScript, "init_routes", 0, null)
+
+    // we need to batch routes as trireme fails to run the script if it's too large
+    routesHelper.routes.sliding(100).foreach { batch =>
+      val script = JavaScriptReverseRouter("batch")(batch: _*)(request).toString
+      cx.evaluateString(scope, s"(function () { $script; mergeDeep(routes, batch); })();", "batch", 0, null);
```

Comment:
Is there anything we can do around assuring the output of the successive mergeDeep calls is the same as the previous implementation?